packages/nimble/R/externalCalls.R

```diff
@@ -0,0 +1,87 @@
+
```

Comment:
Documentation is especially important if you are requesting design review from reviewers.